fromDataUseDomFile is flaky

suspect it was introduced in 903d28f872b90a800c64339dea058806cde0be67.

cc @crowlKats


```
fromDataUseDomFile
AssertionError: Values are not equal:


    [Diff] Actual / Expected


    DomFile {
-     lastModified: 1595350718071,
+     lastModified: 1595350718070,
      name: "bar",
      size: 3,
      type: "text/plain",
      Symbol(bytes): Uint8Array(3) [
        102,
        111,
        111,
      ],
    }

    at assertEquals (file:///Users/rld/src/deno/std/testing/asserts.ts:193:9)
    at fromDataUseDomFile (file:///Users/rld/src/deno/cli/tests/unit/form_data_test.ts:87:3)
    at asyncOpSanitizer (js2/40_testing.js:34:13)
    at Object.resourceSanitizer [as fn] (js2/40_testing.js:68:13)
    at TestRunner.[Symbol.asyncIterator] (js2/40_testing.js:240:24)
    at AsyncGenerator.next (<anonymous>)
    at runTests (js2/40_testing.js:317:22)
    at async workerRunnerMain (file:///Users/rld/src/deno/cli/tests/unit/unit_test_runner.ts:75:3)

Checkfailures:

 file:///Users/rld/src/deno/cli/tests/permission_test.ts
	fromDataUseDomFile
```

--- a/cli/js2/22_form_data.js
+++ b/cli/js2/22_form_data.js
@@ -12,6 +12,7 @@
     if (value instanceof domFile.DomFile) {
       return new domFile.DomFile([value], filename || value.name, {
         type: value.type,
+        lastModified: value.lastModified,
       });
     } else if (value instanceof blob.Blob) {
       return new domFile.DomFile([value], filename || "blob", {

